### ISSUE TITLE
No comments are displayed

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#1143]
STR: 

1. Start Firefox (logged or not logged)
2. go to any issues with comments. Ex: https://webcompat.com/issues/2909 

Actual:
Issues have vanished

Expected:
See the issues.

Have to figure out if it's an issue with webcompat.com or github.

**Comment by karlcow** [comment]
![capture d ecran 2016-07-28 a 12 47 41](https://cloud.githubusercontent.com/assets/505230/17200569/83cebe8c-54c1-11e6-8b2b-0e1a9711f96f.png)

`LinkHeader is null`

**Comment by karlcow** [comment]
`→ http HEAD https://api.github.com/repos/webcompat/web-bugs/issues/2909/comments`


```http
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Cache-Control: public, max-age=60, s-maxage=60
Content-Encoding: gzip
Content-Security-Policy: default-src 'none'
Content-Type: application/json; charset=utf-8
Date: Thu, 28 Jul 2016 04:01:49 GMT
ETag: W/"df396fab6e59210e3b830d207b9a05f3"
Server: GitHub.com
Status: 200 OK
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
Vary: Accept
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-GitHub-Media-Type: github.v3
X-GitHub-Request-Id: 99A27974:4A48:1B653F5:579983AC
X-RateLimit-Limit: 60
X-RateLimit-Remaining: 55
X-RateLimit-Reset: 1469682011
X-Served-By: ef96c2e493b28ffea49b891b085ed2dd
X-XSS-Protection: 1; mode=block
```

Ah maybe they changed their policy to not send a `Link` header when there is only one page. 

Querying all comments of the repos.
`→ http HEAD https://api.github.com/repos/webcompat/web-bugs/issues/comments`


```http
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Cache-Control: public, max-age=60, s-maxage=60
Content-Encoding: gzip
Content-Security-Policy: default-src 'none'
Content-Type: application/json; charset=utf-8
Date: Thu, 28 Jul 2016 04:04:08 GMT
ETag: W/"d6441791b3fbfc1e51dcb4eb89c6b2af"
Link: <https://api.github.com/repositories/17914657/issues/comments?page=2>; rel="next", <https://api.github.com/repositories/17914657/issues/comments?page=323>; rel="last"
Server: GitHub.com
Status: 200 OK
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
Vary: Accept
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-GitHub-Media-Type: github.v3
X-GitHub-Request-Id: 99A27974:4A48:1B692AB:57998437
X-RateLimit-Limit: 60
X-RateLimit-Remaining: 54
X-RateLimit-Reset: 1469682011
X-Served-By: 52437fedc85beec8da3449496900fb9a
X-XSS-Protection: 1; mode=block
```

**Comment by karlcow** [comment]
@miketaylr fun fun fun. That's a day. I guess.

**Comment by karlcow** [comment]
Also it seems there are two functions LinkHeader. Ah probably issue #1118 

https://github.com/webcompat/webcompat.com/blob/f976bc855ac31264ea7d518c4abbc8b2e8585267/webcompat/static/js/lib/models/comment.js#L32

https://github.com/webcompat/webcompat.com/blob/99efc7857238c389c2b401558c961fcc5921f870/webcompat/static/js/lib/models/issue.js#L155


```js
function reqListener () {
  console.log(this.responseText);
}

var oReq = new XMLHttpRequest();
oReq.addEventListener("load", reqListener);
oReq.open("GET", "https://api.github.com/repos/webcompat/web-bugs/issues/2909/comments");
oReq.send();

oReq.getResponseHeader('Link');
# null
```

**Comment by karlcow** [comment]
```
oReq.getResponseHeader('Link').split(',')
```

**Comment by karlcow** [comment]
ok probably 
https://github.com/webcompat/webcompat.com/commit/f976bc855ac31264ea7d518c4abbc8b2e8585267

**Comment by miketaylr** [comment]
TypeError: linkHeader is null
issues.Comment<.parseHeader()
comment.js:43
issues.Comment<.parse()
comment.js:28
.save/r.success()
backbone-min.js:1
m.Callbacks/j()
jquery-1.11.2.min.js:2
m.Callbacks/k.fireWith()
jquery-1.11.2.min.js:2
x()
jquery-1.11.2.min.js:4
.send/b()
jquery-1.11.2.min.js:4
[Learn More]

**Comment by miketaylr** [comment]
Actually. Let's close this because it was fixed by #1144. 

@deepthivenkat please work on this issue back over @ https://github.com/webcompat/webcompat.com/issues/1058 (I've re-opened it), thanks!
